build.gradle

```diff
@@ -25,29 +25,43 @@ group 'bio.terra'
 version '0.0.16-SNAPSHOT'
 sourceCompatibility = JavaVersion.VERSION_1_8
 
+def OPENCENSUS_VERSION = '0.28.2'
```

Comment:
Isaac had some gradle dependency management plugin for syncing versions across multiple libraries. https://github.com/DataBiosphere/terra-workspace-manager/blob/1f4038c4fb69834a3669b054a62db1b284cf26cb/build.gradle#L25-L32
I really don't know what's good practice here, but I hope you're making an informed decision :)

Reply:
The nifty part is the spring version management library is actually already included for spring itself, that's why you don't specify versions for e.g. the spring boot starters.